operations/workplace/README.md

```diff
@@ -4,31 +4,31 @@ description: Description of norms on Mattermost Community server
 
 # Workplace
 
-## What is the Mattermost pre-release server?
+## What is the Mattermost Community Server?
 
 The Mattermost pre-release server runs the latest builds of Mattermost's software, including all open source code for Mattermost Team Edition as well as commercial code for Mattermost Enterprise Edition.
 
 It has three objectives:
 
-* Provide a workplace for Mattermost Staff 
-* Test pre-released builds of our core products
-* Engage with our [communities ](../../contributors/contributors/community.md)including contributors and partners 
+* Provide a workplace for Mattermost staff.
+* Test work-in-progress builds of our core products.
```

Comment:
```suggestion
* Test pre-released builds of our core products.
```

In this case I believe it's fine to say "pre-released" as this is not referring to the server's name.